mobify/platform-scaffold
# Support SVG sprites on AMP

Load svg sprites into the page inline using webpack.

## Related PRs
Merge this first
https://github.com/mobify/platform-scaffold/pull/678

## How to test-drive this PR
- Run `npm i`
- Run `npm run dev`
- Go to `localhost:3000` and check that you can see the user-icon on the homepage.
- View source and convince yourself that the sprite-sheet has been added inline in the HTML.
- Run `npm run build-sprites` and check that it works.
